content/memory_model.tex

```diff
@@ -5,13 +5,13 @@
 \end{figure}      
 %
 An \openshmem program consists of data objects that are private to each \ac{PE}
-and data  objects that are remotely accessible by all \acp{PE}. Private data
+and data  objects that are remotely accessible by all \acp{PE}. Note that the latter 
+of these two data objects are symmetric data objects. Private data
```

Comment:
Instead of "Note that", how about a semicolon?

An \openshmem program consists of data objects that are private to each \ac{PE} and data  objects that are remotely accessible by all \acp{PE}; the latter of these two data objects are +referred to as+ \emph{symmetric data objects}.

Reply:
Works for me. Change has been made.